ir/src/main/scala/org/scalajs/core/ir/Trees.scala

```diff
@@ -193,6 +193,11 @@ object Trees {
     val tpe = NoType // cannot be in expression position
   }
 
+  case class JSForIn(obj: Tree, keyVar: Ident, body: Tree)(
```

Comment:
Given that even `Debugger` is called that way, and not `JSDebugger`, this node should be called `ForIn`.